runtime/doc/options.txt

```diff
@@ -6256,6 +6256,14 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   uselast	If included, jump to the previously used window when
 			jumping to errors with |quickfix| commands.
 
+			*'syncoutput'* *'sync'* *'nosyncoutput'* *'nosync'*
+'syncoutput' 'sync'	boolean	(default off)
```

Comment:
`'sync'` as a short name is a bit non-specific. maybe we can have 'syncout` as the only name (it is just a vimism that options should have a too long name and a too short name, one single name is fully allowed)

Reply:
Changed to `'syncout'` in 144dc9d2cf2584b92b4c4b09363fd9d3304365f7.